framework/src/meshgenerators/FileMeshGenerator.C

```diff
@@ -140,11 +140,11 @@ FileMeshGenerator::deduceCheckpointPath(const MooseObject & object, const std::s
   if (MooseUtils::pathExists(file_name))
     return file_name;
 
-  // xxxx_mesh.cpr -> xxxx-mesh.cpr
-  const std::string old_ending = "_mesh.cpr";
+  // xxxx_mesh.cpa.gz -> xxxx-mesh.cpa.gz
+  const std::string old_ending = "_mesh.cpa.gz";
   if (std::equal(old_ending.rbegin(), old_ending.rend(), file_name.rbegin()))
   {
-    const std::string new_ending = "-mesh.cpr";
+    const std::string new_ending = "-mesh.cpa.gz";
```

Comment:
If we are completely dropping .cpr support, I think we can also drop this prefix renaming support. @loganharbour?